tests/telemetry/events/bgp_events.py

```diff
@@ -9,12 +10,32 @@
 
 
 def test_event(duthost, gnxi_path, ptfhost, data_dir, validate_yang):
-    logger.info("Beginning to test bgp-state event")
+    run_test(duthost, gnxi_path, ptfhost, data_dir, validate_yang, drop_tcp_packets,
+             "bgp_notification.json", "sonic-events-bgp:notification", tag)
     run_test(duthost, gnxi_path, ptfhost, data_dir, validate_yang, shutdown_bgp_neighbors,
              "bgp_state.json", "sonic-events-bgp:bgp-state", tag)
 
 
+def drop_tcp_packets(duthost):
+    logger.info("Adding rule to drop TCP packets to test bgp-notification")
+
+    ret = duthost.shell("iptables -I INPUT -p tcp --dport 179 -j DROP")
+    assert ret["rc"] == 0, "Unable to add DROP rule to iptables"
+
+    ret = duthost.shell("iptables -I INPUT -p tcp --sport 179 -j DROP")
+    assert ret["rc"] == 0, "Unable to add DROP rule to iptables"
+
+    time.sleep(10)  # Give time for hold timer expiry notification to fire
```

Comment:
>hold timer expiry [](http://example.com/codeflow?start=36&length=17)

What is the hold timer expiration time? how 10 is picked? Add some comment to prevent future unexpected magic number change.

Reply:
10 comes from config_db

"BGP_NEIGHBOR": {
        "X": {
            "rrclient": "0",
            "name": "ARISTA02T1",
            "local_addr": "Y",
            "nhopself": "0",
            "admin_status": "up",
            **"holdtime": "10",**
            "asn": "64600",
            "keepalive": "3"
        }

Reply:
Updated comment as well.